FlowCrypt/Controllers/Setup/SetupInitialViewController.swift

```diff
@@ -131,6 +140,28 @@ extension SetupInitialViewController {
         handleCommon(error: error)
         state = .error(error)
     }
+
+    private func checkClientConfigurationIntegrity() {
+        guard let integrityErrorMessage = clientConfigurationService.checkForUsingKeyManager() else {
+            state = .checkingEKMKeys
+            return
+        }
+        showAlert(message: integrityErrorMessage) { [weak self] in
+            self?.router.signOut()
+        }
+    }
+
+    private func checkEKMKeys() {
+        organisationalRulesService.getEmailKeyManagerPrivateKeys()
+            .then { [weak self] result in
```

Comment:
What if there is error pulling keys from EKM (like a network error), where is that handled?